src/components/Autocomplete/Autocomplete.js

```diff
@@ -0,0 +1,382 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import Downshift from 'downshift';
+import memoize from 'fast-memoize';
+import { List, ListItem, ContentPlaceholder, Text, InputWrapper } from '../../index';
+import ItemTag from './ItemTag';
+import styles from './Autocomplete.scss';
+import { NUMBER_OF_SUGGESTION_LOADING_PLACEHOLDERS } from '../../constants';
```

Comment:
shouldn't it be a prop, not a constant?

Reply:
I think it is a constant. We don't have a usecase to overwrite it... If in the future there is, it is really easy to extend it to a prop, and have that prop default to the const.